install_corpora.py

```diff
@@ -0,0 +1,24 @@
+import os.path
+from cltk.corpus.utils.importer import CorpusImporter
+
+print(f'Travis directory for cltk_data is {os.path.expanduser("~/cltk_data/")}')
```

Comment:
Do you mean for this to be in the root of the repo? Could we put it in `cltk/utils`?

Reply:
This is a script specifically for installing corpora (that are not already cached) in travis (cf. 'before_script' in .travis.yml). The idea is to load the corpora that are needed for testing once and at the beginning of the process, and in a way that they can be cached. Experimenting and if this idea isn't ready (or wanted) at the moment—we can close and revisit in the future.

Reply:
About `install_corpora.py` -- I like it. I just think it should moved to `cltk/tests/install_copora.py`.